_content/articles/the-sal-an-alternative-buddhist-holy-tree_ireland-John-D.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "The Sal — An Alternative Buddhist Holy Tree?"
+authors:
+  - "John D. Ireland"
+external_url: "https://journal.equinoxpub.com/BSR/article/view/14508/16725"
+source_url: " https://doi.org/10.1558/bsrv.v17i1.14508"
+drive_links:
+  - "https://drive.google.com/drive/folders/10FhXe3dtyCmt_uQt0VXD8hpMJaDCjS2T"
```

Comment:
Sadly, when you open a file in a Google Drive folder, the URL in the address bar doesn't update. To copy the URL for this file specifically, you'll need to click `…` then "🔗 Get link" then "🔗 Copy link":

![Screenshot_20230420_061938_Chrome](https://user-images.githubusercontent.com/61610173/233219982-4968818d-f5b0-4a1f-8ee3-2824d5342994.jpg)

![Screenshot_20230420_062015_Chrome](https://user-images.githubusercontent.com/61610173/233220010-f341a155-4b87-4bf2-b9b1-ee5b4faae781.jpg)